docs/docs/110-faq.md

```diff
@@ -55,3 +55,19 @@ acorn install --ingress-class-name nginx
 "nginx" is just an example in this case. Your actual ingress class name may vary.
 
 If the endpoint is non-HTTP, like TCP, your cluster needs the ability to support [services of type LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer).
+
+#### I am hitting rate limits from DockerHub
+
+If you are experiencing rate limiting issues from DockerHub, you can log in to your account to increase the number of requests.
+
+```shell
```

Comment:
Dumb question - does `acorn login` not work here?

Reply:
this came from Sangeetha when using --builder-per-project I believe that this is needed b/c the builder doesn't use the creds from acorn login so have to "manually" add it to the namespace at least that is my understanding

Reply:
I wonder if there is a way to get `acorn login` to set this up for a user so they never have to interact with any k8s resources. The only push I have here is that we want to abstract k8s as much as possible for developers just trying to deploy acorns.

Reply:
Agreed, would be cool to add this functionality to acorn login docker.io